Pomodoro.DataAccess/EF/AppDbContextFactory.cs

```diff
@@ -4,29 +4,21 @@
 
 using Microsoft.EntityFrameworkCore.Design;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.Configuration;
 
 namespace Pomodoro.DataAccess.EF
 {
     internal class AppDbContextFactory : IDesignTimeDbContextFactory<AppDbContext>
     {
+        private const string connectionString =
+            @"Data Source=(LocalDB)\MSSQLLocalDB;Initial Catalog=Pomodoro;
```

Comment:
better to move to separate config json file and then inject configs via ConfigurationBuilder for example, like on like 21